keys/printer.go

```diff
@@ -248,53 +266,15 @@ func decodeKeyPrint(key roachpb.Key) string {
 	return buf.String()
 }
 
-// PrettyPrint print the key in a human readable format, which organized as:
-// Key's Format												Key's Value
-// /Local/...												"\x00\x00\x00"+...
-// 		/Store/...											"\x00\x00\x00s"+...
-//		/RangeID/...										"\x00\x00\x00s"+[rangeid]
-//			/[rangeid]/SequenceCache/[id]/seq:[seq]			"\x00\x00\x00s"+[rangeid]+"res-"+[id]+[seq]
-//			/[rangeid]/RaftLeaderLease						"\x00\x00\x00s"+[rangeid]+"rfll"
-//			/[rangeid]/RaftTombstone						"\x00\x00\x00s"+[rangeid]+"rftb"
-//			/[rangeid]/RaftHardState						"\x00\x00\x00s"+[rangeid]+"rfth"
-//			/[rangeid]/RaftAppliedIndex						"\x00\x00\x00s"+[rangeid]+"rfta"
-//			/[rangeid]/RaftLog/logIndex:[logIndex]			"\x00\x00\x00s"+[rangeid]+"rftl"+[logIndex]
-//			/[rangeid]/RaftTruncatedState					"\x00\x00\x00s"+[rangeid]+"rftt"
-//			/[rangeid]/RaftLastIndex						"\x00\x00\x00s"+[rangeid]+"rfti"
-//			/[rangeid]/RangeGCMetadata						"\x00\x00\x00s"+[rangeid]+"rgcm"
-//			/[rangeid]/RangeLastVerificationTimestamp		"\x00\x00\x00s"+[rangeid]+"rlvt"
-//			/[rangeid]/RangeStats							"\x00\x00\x00s"+[rangeid]+"stat"
-//		/Range/...											"\x00\x00\x00k"+...
-//			/RangeDescriptor/[key]							"\x00\x00\x00k"+[key]+"rdsc"
-//			/RangeTreeNode/[key]							"\x00\x00\x00k"+[key]+"rtn-"
-//			/Transaction/addrKey:[key]/id:[id]				"\x00\x00\x00k"+[key]+"txn-"+[id]
-// /Local/Max 												"\x00\x00\x01"
-//
-// /System/...												"\x00"
-//		/Meta1/[key]										"\x00\x00meta1"+[key]
-//		/Meta2/[key]										"\x00\x00meta2"+[key]
-//		/StatusStore/[key]									"\x00status-store-"+[key]
-//		/StatusNode/[key]									"\x00status-node-"+[key]
-// /System/Max												"\x01"
-//
-// /Table/[key]												"\xff"+[key]
-//
-// /Min														""
-// /Max														"\xff\xff"
-func PrettyPrint(key roachpb.Key) string {
-	if bytes.Equal(key, MaxKey) {
-		return "/Max"
-	} else if bytes.Equal(key, MinKey) {
-		return "/Min"
-	}
-
+func prettyPrintInternal(key roachpb.Key) (string, bool) {
 	var buf bytes.Buffer
+	formatted := true
```

Comment:
You're never setting this variable to a different value. I'd just remove it and return `true` in the places it is being used.